src/main/antlr/LibSLLexer.g4

```diff
@@ -254,6 +254,43 @@ NULL
    :   'null'
    ;
 
+IntegerLiteral:
+    DecimalIntegerLiteral
+    | HexIntegerLiteral
+    | OctalIntegerLiteral
+    | BinaryIntegerLiteral
+;
+
+fragment DecimalIntegerLiteral: DecimalNumeral IntegerTypeSuffix?;
+
+fragment HexIntegerLiteral: HexNumeral IntegerTypeSuffix?;
+
+fragment OctalIntegerLiteral: OctalNumeral IntegerTypeSuffix?;
+
+fragment BinaryIntegerLiteral: BinaryNumeral IntegerTypeSuffix?;
+
+fragment DecimalNumeral: '0' | NonZeroDigit (Digits?);
+
+fragment IntegerTypeSuffix: [lL];
```

Comment:
May be we should extend this "list" a bit.

Reply:
Fixed )